### DISCUSSION TITLE
集群调度系统api

### DISCUSSION THREAD

**Discussion by knight134** [skyformaip/skyformaip#7] [Q&A]
请问贵方集群调度软件是否提供api。
在运行生信流程时，想通过api进一步将任务的投递-监控和告警自动化。

**Comment by skyformaip** [comment]
> 请问贵方集群调度软件是否提供api。 在运行生信流程时，想通过api进一步将任务的投递-监控和告警自动化。

REST API在企业版中支持。如有进一步的需求，请联系我们。
